tests/test_md_cli.py

```diff
@@ -40,7 +40,10 @@ def test_md_help():
 def test_md(ensemble, tmp_path):
     """Test all MD simulations are able to run."""
     file_prefix = tmp_path / f"{ensemble}-T300"
-    traj_path = tmp_path / f"{ensemble}-T300-traj.xyz"
+    if ensemble in ["npt", "nph"]:
+        traj_path = tmp_path / f"{ensemble}-T300-{ensemble}-T300.0-p0.0-traj.xyz"
```

Comment:
Slightly concerning that these are both rounded or unrounded seemingly inconsistently. It may result in unintentional overwriting.